src/styles/components/index.js

```diff
@@ -142,9 +146,72 @@ export const AboutImg = styled.img`
         padding: 0 2rem;
     }
 `
+export const SponsContent = styled.div`
```

Comment:
can we update these variables to be `Spon-` instead? I think we're gonna unify `spon` = sponsor-related on API too

Reply:
Just gonna make this a bug to fix later

Reply:
HCK-37

Reply:
amazing